pkg/api/v1/pod/util_test.go

```diff
@@ -296,12 +322,13 @@ func TestVisitContainers(t *testing.T) {
 				},
 			},
 			[]string{"i1", "i2", "c1", "c2", "e1", "e2"},
+			AllContainers,
 		},
 	}
 
 	for _, tc := range testCases {
```

Comment:
Wrap inner block in a `t.Run(tc.description,`. Since you're already in here, let's clean this up.

Reply:
done